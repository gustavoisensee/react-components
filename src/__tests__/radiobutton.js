import React from 'react';
import renderer from 'react-test-renderer';
import Radiobutton from './../modules/radiobutton/component';

describe('Radiobutton component test', () => {
  test('Radiobutton component renders', () => {
    const props = {
      id: '',
      name: '',
      text: '',
      value: 1,
      selected: 1,
      extraClass: '',
      change: () => {}
    };

    const tree = renderer.create(
      <Radiobutton {...props} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
