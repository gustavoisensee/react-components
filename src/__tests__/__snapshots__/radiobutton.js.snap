exports[`Radiobutton component test Radiobutton component renders 1`] = `
<div
  className="filter">
  <input
    checked={true}
    className="radiobutton"
    id=""
    name=""
    onChange={[Function]}
    type="radio"
    value={1} />
  <label
    className="radiobutton__label"
    htmlFor="">
    
  </label>
</div>
`;
